htdocs/edittags.bml

```diff
@@ -103,8 +103,11 @@ body<=
     $ret .= "<script type='text/javascript'> var cur_taglist = '$logtagstr'; </script>";
 
     $ret .= '<table summary="" class="edittbl" cellpadding="0" cellspacing="0" width="50%">';
+    if ( $u->is_community ) {
+        $ret .= "<tr><td class='l'>$ML{'.poster'}</td><td>" . $ent->poster->ljuser_display . "</td></tr>"
+        };
     $ret .= "<tr><td class='l'>$ML{'.subject'}</td><td>$subj</td></tr>" if $subj;
-
+    
```

Comment:
I think what happened was a blank line (e.g. `\n`) was replaced with a line that contains whitespace indenting the line to the level of the line above (e.g. `\s\s\s\s\n`) - it should ideally be completely unchanged.  These creep in from time to time - if it's too troublesome don't worry about it!